realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -462,6 +477,15 @@ private static String getManagementRealmUrl(URL authUrl) {
         }
     }
 
+    /**
+     * FIXME Javadoc
+     *
+     * @return
+     */
+    public PermissionManager getPermissionManager() {
```

Comment:
What is the designed way for user to get the `PermissionManager`? through this API or the constructor? It is confusing to me if the `close()` should be called on the return value of this API.